pkg/thethingsgateway/cups/handlers.go

```diff
@@ -128,9 +144,15 @@ func adaptGatewayAddress(address string) (result string, err error) {
 		host = address
 	}
 	if port == "" {
-		port = "8882"
+		port = "8881"
+	}
+	switch port {
```

Comment:
Maybe default to `mqtts` ? Otherwise custom ports will end up with an invalid `://host:port` format.